client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx

```diff
@@ -5,10 +5,10 @@ import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
 const WPCourses = () => {
 	return (
 		<Task
-			title="Introducing WordPress.com Courses"
-			description="Build your skills with in-depth courses for any level, taught by WordPress experts. Enroll to access on-demand content, a private online community, office hours, and virtual meetups."
-			actionText="Enroll today!"
-			actionUrl="https://wpcourses.com/?utm_source=wordpressdotcom&utm_medium=referral&utm_campaign=courses_launch_myhome"
+			title="World-class education by WordPress&nbsp;experts"
```

Comment:
We can `import { preventWidows } from 'calypso/lib/formatting'` to do the nbsp trick more semantically. And it means the nbsp won't get submitted to translators.

Reply:
This won't get translated as the card is only available to English locales (https://github.com/Automattic/wp-calypso/pull/55684). I have a weak preference for the simplicity of adding it inline but happy to change if we generally prefer using `preventWidows`.

Thanks for adding the import to the suggestion, that would have saved a step for me!

Reply:
Makes sense. Non-translated text doesn't come up often enough that I think we have a guideline about preferring `preventWidows`. I'm happy to leave the nbsp inline 👍